.jest/env.js

```diff
@@ -0,0 +1,2 @@
+import { tmpdir } from 'os'
+process.env.CONTENTFUL_CONFIG_FILE = `${tmpdir()}/.contentfulrc.json`
```

Comment:
```suggestion
var path = require('path');

process.env.CONTENTFUL_CONFIG_FILE = path.join(tmpdir(), '.contentfulrc.json');
```

let's also make the windows friends happy :)